src/handler/operation/cookie.js

```diff
@@ -67,9 +68,13 @@ function tabToStringUrl(tab) {
   return `${url.protocol}//${url.hostname}`;
 }
 
-function tabToStringDomain(tab) {
-  const url = new URL(tab.url);
+function tabToStringDomain(inputUrl) {
+  const url = new URL(inputUrl.startsWith("http") ? inputUrl : `http://${inputUrl}`);
```

Comment:
just a suggestion of setting a const "http://" and use it twice here, allowing some (although minor) reuse. Would assume no issue when https is used

Reply:
In this case, we are just concatenating HTTP to form a valid URL in order to validate it with the URL class, so no hard in just doing HTTP.
It makes sense, I will add it as a constant.